src/pudl/metadata/fields.py

```diff
@@ -1120,6 +1120,15 @@
         "description": "Electricity demand calculated by subtracting BA interchange from net generation, with outliers and missing values imputed by EIA.",
         "unit": "MWh",
     },
+    "demand_imputed_pudl_mwh": {
+        "type": "number",
+        "description": "Electricity demand calculated by subtracting BA interchange from net generation, with outliers and missing values imputed by in PUDL.",
+        "unit": "MWh",
+    },
+    "demand_imputed_pudl_mwh_imputation_code": {
+        "type": "string",
+        "description": "Code describing why a demand value was flagged for imputation.",
```

Comment:
I think this field needs to have an `enum` constraint applied to it if we want it to be treated as a categorical value and dictionary encoded in Parquet -- otherwise I think it'll treat every string as potentially unique and store them all, which will take up much more memory / disk than necessary.

Reply:
Added